docs/docsite/.templates/banner.html

```diff
@@ -61,14 +55,11 @@
       }
       msg += '</p>';
 
-      banner += '<div id="banner_id" class="admonition ';
-      banner += important ? 'important' : 'caution';
-      banner += '">';
+      banner += '<div id="banner_id" class="admonition '; banner += important ? 'important' : 'caution'; banner += '">';
```

Comment:
I assume this is an accidental change?
```suggestion
      banner += '<div id="banner_id" class="admonition ';
      banner += important ? 'important' : 'caution';
      banner += '">';
```